tests/flow/test_config.py

```
@@ -172,6 +173,22 @@ def test07_config_invalid_subcommand(self):
             assert("Unknown subcommand for GRAPH.CONFIG" in str(e))
             pass
 
+    def test12_index_worker_threads_immutable(self):
+        # INDEX_WORKER_THREADS is a load-time only configuration; it must
+        # default to 0 and reject runtime GRAPH.CONFIG SET.
+        response = self.db.config_get("INDEX_WORKER_THREADS")
+        self.env.assertEqual(response, 0)
+
+        try:
+            self.redis_con.execute_command("GRAPH.CONFIG SET INDEX_WORKER_THREADS 8")
+            assert(False)
```

Ruff (0.15.15)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)